### PR TITLE
Fix 235

### LINKED ISSUE #235: format_vehicle and format_casualty
Hi:

If I download and read in the full CSV (e.g., https://data.dft.gov.uk/road-accidents-safety-data/dft-road-casualty-statistics-casualty-1979-latest-published-year.csv) using read.csv, and then I use format_casualty or format_vehicle on that df, it turns age of casualty and age of vehicle to NA, and leaves only descriptions of those results that are actually NA.

Ivo

### PR DESCRIPTION
**Pull request by layik**
- Change variable csv for only few variables setting them to integer classes
- Add few lines of code to feed `readr::read_csv` with `stats19_variables$type` classes on read
- Prevent `format` function changing column types of the tibble/df's read in 

Checks?
Does this work with all types of objects supported by stats19?

closes #235

### CODE REVIEW

**Comment by layik** [comment]
ToDo:
- [ ] make Windows happy with vignetters as above
- [ ] Address #220 where if age_of_casualty is -1 or -3 then it should not be converted to "Data out of range" like text. Perhaps part of #220 should come here or the two should be combined.

I believe that is all needed to press "Ready for review"

**Comment by Robinlovelace** [comment]
Looking *really* good. Many thanks Layik :pray:

**Comment by layik** [comment]
On a Win instance the issue seems to be at a simple get_stats19 call, Windows complains about invalid parameter type which makes no sense.

Any help would be appreciated as the CI gives nothing to work with.

File: data-raw/stats19_variables.csv [hunk 1]
```diff
@@ -43,7 +43,7 @@ Casualty,accident_index,unique value for each accident. The accident_index combi
 Casualty,accident_reference,"In year id used by the police to reference a collision. It is not unique outside of the year, use accident_index for linking to other years",accident_reference,character
 Casualty,accident_year,NA,accident_year,numeric
 Casualty,age_band_of_casualty,NA,age_band_of_casualty,character
-Casualty,age_of_casualty,NA,age_of_casualty,character
+Casualty,age_of_casualty,NA,age_of_casualty,numeric
```
Comment (Robinlovelace) [inline data-raw/stats19_variables.csv:46]: 👍

File: R/read.R [hunk 2]
```diff
@@ -169,5 +169,30 @@ read_ve_ca = function(path) {
 
 read_null = function(path, ...) {
   if(is.null(path)) return(NULL)
-  readr::read_csv(path, ...)
+  readr::read_csv(path, col_types = col_spec, ...)
 }
+
+# possibly in utils
+# Convert the 'type' column to readr's col_type format
+convert_to_col_type <- function(type) {
```
Comment (Robinlovelace) [inline R/read.R:177]:
```suggestion
convert_to_col_type <- function(type) {
```

**Review by Robinlovelace — COMMENTED** [review]
Minor suggestion in the review, let's get this merged!

File: R/read.R (hunk 2 above)
Comment (layik) [reply R/read.R]: should have used the GH option :)

**Comment by Robinlovelace** [comment]
Awesome, thank you Layik!

**Pull request merged**